libvcx/src/connection.rs

```diff
@@ -295,38 +295,38 @@ pub mod tests {
         handle
     }
 
-    pub fn create_connected_connections(consumer_handle: Option<u32>) -> (u32, u32) {
+    pub fn create_connected_connections(consumer_handle: Option<u32>, institution_handle: Option<u32>) -> (u32, u32) {
         debug!("Institution is going to create connection.");
-        ::utils::devsetup::set_institution();
-        let faber_to_alice = create_connection("alice").unwrap();
+        ::utils::devsetup::set_institution(institution_handle);
+        let institution_to_consumer = create_connection("consumer").unwrap();
         let _my_public_did = settings::get_config_value(settings::CONFIG_INSTITUTION_DID).unwrap();
-        let details = connect(faber_to_alice).unwrap().unwrap();
-        update_state(faber_to_alice).unwrap();
+        let details = connect(institution_to_consumer).unwrap().unwrap();
+        update_state(institution_to_consumer).unwrap();
```

Comment:
I believe this `update_state` doesn't perform any action. After `connect` is called, `agent_connection` is created in agency and next `update_state` only checks whether the counterparty has sent connection request message.

Reply:
You are right, good catch!